packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -150,6 +149,12 @@ Template.AssistifyCreateRequest.onRendered(function() {
 
 		return instance.find('.js-save-request').focus();
 	});
+
+	if (this.find('input[name="expertise"]').value) {
+		this.find('input[name="request_title"]').focus();
+	} else {
+		this.find('input[name="expertise"]').focus();
```

Comment:
For performance reasons it might be good calling `find` on `'input[name="expertise"]'` only once.

Reply:
thanks, fixed